src/eventkit.cs

```diff
@@ -508,16 +508,14 @@ interface EKEventStore {
 
 		[Export ("defaultCalendarForNewEvents"), NullAllowed]
 		EKCalendar DefaultCalendarForNewEvents { get;  }
-
-#if !MONOMAC
-		[NoWatch]
+		
+		[NoWatch, Mac (10,14)]
```

Comment:
Tests, run on 10.14, failed - that shows that this did not exists back then
Please update to `[Mac (10,15)]`